tests/RobotFramework/tests/cumulocity/mapper_recovery/recover_and_publish_software_update_message.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:mapper recovery 
+Suite Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+Mapper recovers and processes output of ongoing software update request
+    # Mapper receieves a software update request
```

Comment:
These comments should be in a `[Documentation]` block so that they are visible in the RobotFramework report. You can also write a multiline doc block like so (an example taken from tests/RobotFramework/tests/customizing/config_dir.robot):

```
[Documentation]    Check a thin-edge.io executable by running it for a short time (~5s), and 
    ...                analyzing the standard error output for any signs of the default
    ...                path /etc/tedge
    ...                This is only a rough check, as there is not a clean way to check
    ...                the current configuration of a specific component
```

Reply:
fixed